packages/frontend/src/components/Settings/Profile.tsx

```diff
@@ -14,7 +14,7 @@ type Props = {
 
 export default function Profile({ settingsStore }: Props) {
   const initials = avatarInitial(
-    settingsStore.settings.displayname || '',
+    (settingsStore.settings && settingsStore.settings.displayname) || '',
```

Comment:
wouldn't be 
```suggestion
    (settingsStore.settings?.displayname) || '',
```
sufficient here?

Reply:
It certainly would. Done